### PR TITLE
Item list

### PR DESCRIPTION
**Pull request by bsonge**
ListView:  Pass in an array of objects, it picks out the first image it finds and plops it in the left-hand side or displays an 'image not found' image.  Displays data in a tidy list.

Future goals: Pages?

### CODE REVIEW

**Review by Quinn-Donnelly — CHANGES_REQUESTED** [review]
Investigate bug we added it to the search page and passed it the same array that was being past to the ItemTable. When this was done the list was generated but was not showing data.

File: app/components/ItemList/index.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      if (obj[key] && obj[key].toString().slice(0, 4) === 'http') {
+        pic = obj[key];
+        picKeys.push(key);
+      } else if (!picKeys.includes(key)) { // don't display keys proven to be pics
+        items.push(
+          <Table style={{ padding: 0, margin: 0 }}>
+            <tbody>
+              <tr>
+                <td style={{ width: '30%' }}><strong>{key}:</strong></td>
+                <td>{obj[key] || ''}</td>
```
Comment (wrejcodes) [inline app/components/ItemList/index.js:42]: There is a bug around here preventing the data to be displayed. I was able to get it to display the data by moving  the stuff within braces from line 43 to right after the closing strong tag on line 42. Need further validation that this fixes it

**Review by wrejcodes — CHANGES_REQUESTED** [review]
After these issues are fixed I think it will be good.

**Review by Jubstron — APPROVED** [review]
I don't have the component to test. So I hesitate to approve this, but I don't see any glaring issues at the moment

**Comment by Quinn-Donnelly** [comment]
Merging so i can work with it on my end since the files don;t change pages shouldn't break server.

**Pull request merged**